lib/akamai_rspec/matchers/non_akamai.rb

```diff
@@ -42,8 +42,15 @@ module Matchers
         rescue RestClient::MaxRedirectsReached
           return true # Securely sent a redirect
         rescue Exception => e
-          @error = e
-          return false
+          begin
+            if e.response # Securely sent a non-success HTTP code
+                          # See: https://github.com/rest-client/rest-client/tree/1.8.x#exceptions-see-wwww3orgprotocolsrfc2616rfc2616-sec10html
```

Comment:
Fair cop.

This is easier in rest-client v2.0 ...   but on v1.8, this approach is documented [here](https://github.com/rest-client/rest-client/tree/1.8.x#exceptions-see-wwww3orgprotocolsrfc2616rfc2616-sec10html) - so I think it should be okay.